packages/contractkit/src/wrappers/Exchange.test.ts

```diff
@@ -0,0 +1,55 @@
+import { newKitFromWeb3 } from '../kit'
+import { testWithGanache } from '../test-utils/ganache-test'
+import { ExchangeWrapper } from './Exchange'
+
+/*
+TEST NOTES:
+- In migrations: The only account that has cUSD is accounts[0]
+*/
+
+testWithGanache('Exchange Wrapper', (web3) => {
+  const ONE = web3.utils.toWei('1', 'ether')
```

Comment:
Style nit: I think we typically don't ALL_CAPS TypeScript constants.

Reply:
not sure about this... @jmrossy is this correct?